index.html

```diff
@@ -41,20 +42,24 @@ <h1>Happy Hearts Roaming</h1>
           <h1>My story</h1>
           <div>
             <p>
-              I am Minna, an aspiring software developer from Finland and a
-              proud mom of a spirited 3-year-old. I feel fortunate to have my
-              parents, who are still healthy, from Vietnam, along with my
-              beloved sister from the US, who is not only very close to me but
-              also shares my passion for traveling together. Our family
-              frequently travels across continents to reconnect and create
-              cherished memories together...
+              Lorem ipsum odor amet, consectetuer adipiscing elit. Netus
+              habitant quam elit adipiscing tempor consectetur eget mattis.
+              Donec hac dapibus arcu auctor gravida curae vel. Commodo tellus
+              praesent eleifend feugiat curae. Quisque eget laoreet ridiculus
+              suspendisse sollicitudin aliquam magna felis. Vel laoreet potenti
+              libero finibus sodales in tristique. Velit et dignissim torquent
+              faucibus nunc. Egestas leo turpis feugiat volutpat felis...
             </p>
           </div>
           <a class="readmore-button" href="mystory.html">
             <button>READ MORE</button>
           </a>
         </div>
-        <img class="profile-pic" src="images/image3.jpg" alt="" />
+        <img class="profile-pic" src="https://via.placeholder.com/300" alt="" />
```

Comment:
This image is not working

Reply:
I will replace it with actual images to better reflect how the design will appear